apng-drawable/src/main/kotlin/com/linecorp/apng/ApngDrawable.kt

```diff
@@ -280,11 +283,44 @@ class ApngDrawable @VisibleForTesting internal constructor(
     }
 
     private fun computeBitmapSize() {
-        scaledWidth = scaleFromDensity(width, sourceDensity, targetDensity)
-        scaledHeight = scaleFromDensity(height, sourceDensity, targetDensity)
+        scaledWidth = scaleFromDensity(apngState.width, apngState.sourceDensity, targetDensity)
+        scaledHeight = scaleFromDensity(apngState.height, apngState.sourceDensity, targetDensity)
         dstRect.set(0, 0, scaledWidth, scaledHeight)
     }
 
+    internal class ApngState(
```

Comment:
Please move this class to a single class file.

Reply:
ApngDrawable has internal constructor with ApngState as parameter so it is better to keep it as it is.